README.rst

```diff
@@ -512,18 +514,18 @@ only those exact fixers are enabled whether or not level is set.
 With the ``--config-file`` option you can specify the path to the
 ``.php_cs`` file.
 
-By using ``--using-cache`` option you can set if caching
-mechanism should be used.
+By using ``--using-cache`` option with ``yes`` or ``no`` you can set if caching
+mechanism should be used (enabled by default).
```

Comment:
I tried to suggest that the caching is enabled by default (as was in the docs), but tried by adding the ``yes`` and ``no`` explaining how to enable or disable this default (which is missing in the docs). I think that the ``yes`` and ``no`` description should be in because ``1`` vs. ``0`` or ``Y`` vs. ``N`` etc. make as much sense . Which leaves the end-user guessing ``no``.
Maybe I tried to write the docs with a different reading in mind. Do you have a suggestion how to adjust the docs?